.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we should probably add .classpath to .ignore file just to prevent complications.